observation_portal/proposals/templates/proposals/timeallocationreminder.html

```diff
@@ -0,0 +1,50 @@
+{% load i18n %}
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Time Allocation Notification</title>
+</head>
+
+<body>
+<p>
+Dear {{ proposal.pi.first_name}} {{ proposal.pi.last_name }},
+</p>
+
+<p>
+The following table shows your network time allocation and usage of {{ proposal }} so far (through {% now "F j, Y" %}) for
+{{ proposal.current_semester }}.  Note that the semester ends on
+{{ proposal.current_semester.end|date:"F j, Y" }}, and unused time does not carry over.
+Also, be aware that the end of the semester generally sees higher contention for observing resources.
+</p>
+
+<table cellpadding=10>
+  <thead>
+    <tr>
+      <th></th><th colspan="3">Hours Allocated</th><th colspan="3">Hours Used</th>
+    </tr>
+    <tr>
+      <th></th><th>Queue</th><th>Time Critical</th><th>Rapid Response</th>
+      <th>Queue</th><th>Time Critical</th><th>Rapid Response</th>
```

Comment:
Should these column headers have different names (like Queue Allocated, Queue Used)?

Reply:
There are 2 rows of headers, the top one says Hours Allocated, Hours Used. It looks good if you actually see it. There is a screenshot on the pivotal story.

Reply:
Oh your right, I was having trouble visualizing it. It looks fine then